code/modules/mob/living/living.dm

```diff
@@ -1139,6 +1139,8 @@
 		if("eye_blurry")
 			set_blurriness(var_value)
 		if("maxHealth")
+			if (!isnum(var_value) || var_value <= 0)
+				return FALSE
```

Comment:
pretty sure this is too late, `..()` has already been called

Reply:
wait fuck